src/components/Lists/List.vue

```diff
@@ -266,6 +266,10 @@ export default {
       display: grid;
     }
 
+    @media($small) {
+      scroll-snap-align: center;
```

Comment:
I'm not sure if this is working, I may work on adding snap horizontally as well so it snaps to the whole list column